src/nunit.analyzers.tests/TestMethodUsage/TestMethodUsageAnalyzerTests.cs

```diff
@@ -510,6 +510,7 @@ public void AnalyzeWhenSimpleTestMethodHasParameters()
 
             var testCode = TestUtility.WrapMethodInClassNamespaceAndAddUsings(@"
     [↓Test]
+    [Category(""Test"")]
```

Comment:
Previously there was a bug that diagnostic was added to non-test attributes in this case.